doc/releases/octopus.rst

```diff
@@ -1,3 +1,16 @@
+v15.2.7 Octopus
+===============
+
+This is the 7th backport release in the Octopus series. This release fixes
+a serious bug in RGW garbage collection that has been shown to cause data loss
+under certain circumstances.
```

Comment:
it's probably a good idea to describe the 'certain circumstances' more clearly, so users can better evaluate the risk. @ivancich provided some good text on the ceph-users list:

> This bug is triggered when a read of a large RGW object (i.e., one with at least one tail segment) takes longer than 1/2 (one half) the time specified in the configuration option rgw_gc_obj_min_wait (2 hours by default, specified in seconds). The bug causes the tail segments of that read object to be added to the RGW garbage collection queue, which will in turn cause them to be deleted after a period of time.

Reply:
@cbodley Done. I omitted the parenthetical statements because it seems to read better without them.